static/js/settings_org.js

```diff
@@ -184,6 +184,15 @@ function get_property_value(property_name) {
         return JSON.stringify(page_params[property_name]);
     }
 
+    if (property_name === 'realm_oss_showcase_policy') {
+        if (page_params.realm_oss_showcase_policy === 1) {
+            return "listed";
+        }
+        if (page_params.realm_oss_showcase_policy === 2) {
+            return "hidden";
+        }
+    }
+
```

Comment:
We do not require this, as now we want to set the dropdown value as integer only, not these strings.